CHANGELOG.md

```diff
@@ -16,6 +16,37 @@ Also, any bug fix must start with the prefix �Bug fix:� followed by the desc
 
 Previous classification is not required if changes are simple or all belong to the same category.
 
+## [8.1.6]
```

Comment:
This is not correct. We are still in version 8.1.5. Move this information to the 8.1.5 notes.

Mind the changes in Semantic Kernel.